engine/src/main/antlr4/org/hawkular/alerts/engine/tags/parser/TagQuery.g4

```diff
@@ -127,4 +127,6 @@ fragment V: [vV];
 fragment W: [wW];
 fragment X: [xX];
 fragment Y: [yY];
-fragment Z: [zZ];
\ No newline at end of file
+fragment Z: [zZ];
+
+fragment NEG_OP: '~';
```

Comment:
In the current tagQuery language http://www.hawkular.org/docs/rest/rest-alerts.html a <tag_value> can include a regexp if it is defined between simple quotes.
This '~' char is a potential code to be escaoed in examples like:
```
def tagQuery = "test_tag = '\\/t;hawkular\\/f;my-agent\\/r;Local%20DMR\\~\\~_Server Availability'"
```
And it was not supported by the lexer.
This one is fixed as it might look like a corner case but it is common in for Hawkular Services.